### ISSUE TITLE
Strange artifacts in drawings

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#5531]
When you run the `FLOOR PLAN - ENLARGED - 1ST FLOOR` drawing sometimes it works and sometimes it has these strange artifacts. See GIF for diff.

files: 
[drawing artifacts.zip](https://github.com/user-attachments/files/17248573/drawing.artifacts.zip)

![2024-10-03_13-34-12_chromeFLOOR_PLAN_-_ENLARGED_-_1ST_FLOOR svg_-_Google_Chrchrome](https://github.com/user-attachments/assets/aafc6471-9130-4c77-8bae-8715ff3caceb)

**Comment by Moult** [comment]
I can't seem to reproduce this. It regularly looks like this to me, which matches what's in the viewport:

![Image](https://github.com/user-attachments/assets/fdafb4ae-97dd-4a24-8e5a-a7dcff7e3728)

**Comment by sboddy** [comment]
I'd also add that opening the file shows that the windows have been shifted away from where the openings are. Whether this was operator error or some code change that affected the file I don't know:
![Image](https://github.com/user-attachments/assets/0d635e08-085d-4e70-a358-d3e382db0a94)

Looking at the XML in Inkscape, the same ifc:guid (i.e. 04CvAA0HT27B1whX4_EKKj) has two groups, a projection (product-0433928a-0117-421c-b07a-ae113e39452d which is shifted left like the windows in the 3D view above) and a cut (g1538 which is in the correct position in theoryshaws images). In mine and Moults screenshots, the cut and project are aligned properly - just the windows are in the wrong position.

**Comment by theoryshaw** [comment]
yeah, not sure how these windows shifted over.  I'm closing (to save on resources) until if/when I can consistently replicate again.
